src/config.ini

```diff
@@ -24,9 +24,10 @@ num_faculties=14
 num_courses=313
 num_course_units=6513
 num_course_metadata=8590
-num_schedules=11120
-num_schedule_professor=42737
-num_professors=2688
+num_classes=11000
+num_slots=21000
+num_slot_professor=40000
+num_professors=999999
```

Comment:
This numbers are too high, maybe it would be better to lower them so it doesn't appear `0%` while scrapping the professors because of the specified number which is way larger than the actual number of professors.